tools/lib.sh

```diff
@@ -75,3 +75,7 @@ function contains_element() {
   return 1
 }
 
+function validate_config() {
+  log "Validating json config file $1"
+  python -m json.tool $1 1>/dev/null 
```

Comment:
Bah, the reason why we put that into osqueryd was so that it could be used in situations like this, but after reading through initialization code, you're right. We could move the config check into another binary. osqueryi maybe?